server/portal/settings/settings_secret.example.py

```diff
@@ -225,11 +231,32 @@
     _PORTAL_DATA_DEPOT_PROJECTS_SYSTEM_PREFIX
 )
 _PORTAL_PROJECTS_ROOT_HOST = 'cloud.corral.tacc.utexas.edu'
-_PORTAL_PROJECTS_PRIVATE_KEY = ''
-_PORTAL_PROJECTS_PUBLIC_KEY = ''
 _PORTAL_PROJECTS_FS_EXEC_SYSTEM_ID = ''
 _PORTAL_PROJECTS_PEMS_APP_ID = ''
-_PORTAL_USER_ACCOUNT_SETUP_WEBHOOK_PWD = '123'
+_PORTAL_PROJECTS_PRIVATE_KEY = ''
+_PORTAL_PROJECTS_PUBLIC_KEY = ''
+
+########################
+# Custom Portal Template Assets
+# Asset path root is static files output dir.
+# {% static %} won't work in conjunction with {{ VARIABLE }} so use full paths.
+########################
+
+# No Art.
+# _PORTAL_ICON_FILENAME=''                 # Empty string yields NO icon.
+
```

Comment:
I vote kill this comment. Force a value. We'll deal with favicon in FP-263. I'm running out of brain power. Tell me if I am "off my rocker".

```suggestion
```

Reply:
Fine by me